### ISSUE TITLE
task03: other classification methods

### ISSUE THREAD

**Issue by Revilllo7** [Revilllo7/computational-intelligence-class#34]
KNN for k=3, k=5, k=11

Naive Bayes classification, MLP classification

1. split the dataset into 70/30 training and testing
2. run each classificator using packages and evaluate the testing set, print the accuracy and confusion matrix
(google: sklearn scikit + classificator name)
[supervised learning scikit](https://scikit-learn.org/stable/supervised_learning.html)
3. compare all 5 classificators:
- decTree
- 3NN
- 5NN
- 11NN
- NB
- MLP
4. write a report about findings
